cg/meta/workflow/balsamic.py

```diff
@@ -94,7 +94,7 @@ def get_case_path(self, case_id: str) -> Path:
 
     def get_cases_to_analyze(self) -> list[Case]:
         cases_query: list[Case] = self.status_db.cases_to_analyze(
-            workflow=self.workflow, threshold=self.use_read_count_threshold
+            workflow=self.workflow, threshold=self.use_read_count_threshold, limit=21
```

Comment:
We should define the values as constant, same as we're doing with `MAX_ITEMS_TO_RETRIEVE`

Reply:
Done